transactions/README.md

```diff
@@ -60,64 +60,62 @@ fileName:transaction.json
+  <span class="hljs-attr">"countryCode"</span>: <span class="hljs-string">"GBR"</span>,
+  <span class="hljs-attr">"scheme"</span>: <span class="hljs-string">"visa"</span>,
+  <span class="hljs-attr">"firstNumbers"</span>: <span class="hljs-string">"555500"</span>,
+  <span class="hljs-attr">"lastNumbers"</span>: <span class="hljs-string">"5001"</span>,
+  <span class="hljs-attr">"address"</span>: <span class="hljs-string">"2 Soho Square"</span>,
+  <span class="hljs-attr">"postcode"</span>: <span class="hljs-string">"W1D3PX"</span>,
+  <span class="hljs-attr">"city"</span>: <span class="hljs-string">"London"</span>,
+  <span class="hljs-attr">"merchantId"</span>: <span class="hljs-string">"12345"</span>,
+  <span class="hljs-attr">"live"</span>: <span class="hljs-literal">false</span>,
+  <span class="hljs-attr">"auth"</span>: <span class="hljs-literal">true</span>,
+  <span class="hljs-attr">"cleared"</span>: <span class="hljs-literal">true</span>,
+  <span class="hljs-attr">"time"</span>: <span class="hljs-string">"2017-03-02T19:12:01.743Z"</span>,
+  <span class="hljs-attr">"date"</span>: <span class="hljs-string">"2017-03-02T19:12:01.743Z"</span>,
+  <span class="hljs-attr">"created"</span>: <span class="hljs-string">"2017-03-02T19:12:01.744Z"</span>,
+  <span class="hljs-attr">"updated"</span>: <span class="hljs-string">"2017-03-02T19:12:01.744Z"</span>,
+  <span class="hljs-attr">"offer"</span>: <span class="hljs-literal">null</span>,
+  <span class="hljs-attr">"medatada"</span>: {
+    <span class="hljs-attr">"id"</span>: <span class="hljs-string">"your-unique-id"</span>,
+    <span class="hljs-attr">"property"</span>: <span class="hljs-string">"value"</span>
   }
-}
-```
+}</code><span class="line-numbers-rows" aria-hidden="true"><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span></span></pre></div>
+</details>
 
-There are two types of transactions depending on the time of processing and clearing state: authorization transactions and clearing transactions. Authorization transactions are processed in real-time, when the user pays in-store (only available on MasterCard and American Express. Please email [developer@fidel.uk](mailto:developer@fidel.uk) for VISA availability). You can use the `transaction.auth` webhook event to notify or reward the user in your application in real-time.
+There are two types of Transactions depending on the time of processing and clearing state: authorization Transactions and clearing Transactions. Authorization Transactions are processed in real-time. You can use the `transaction.auth` webhook event to notify or reward the user in your application in real-time.
```

Comment:
the 2 instances of `real-time` are a bit too close to each-other, maybe we can omit the 2nd one, wdyt?